inc/core/class-astra-attr.php

```diff
@@ -72,7 +72,12 @@ public function astra_attr( $context, $attributes = array(), $args = array() ) {
 				if ( true === $value ) {
 					$output .= esc_html( $key ) . ' ';
 				} else {
-					$output .= sprintf( '%s="%s" ', esc_html( $key ), esc_attr( $value ) );
+					// Added image itemprop condition for schema markup.
+					if ( 'itemprop' === $key && 'image' === $value ) {
```

Comment:
```suggestion
					if ( 'itemprop' === $key && 'image' === $value ) {
```
@GauravKhupse This condition should be dynamic. If in the future, we add any other string instead of image, this condition will fail. Please maybe try escaping the quotes if double qoutes are present in the string.

Reply:
@premanshup We will discuss.